extras/nest_vars.sh.in

```diff
@@ -1,26 +1,19 @@
 #!/bin/sh
 
-# NEST is installed here. When you relocate NEST, change this variable.
-export NEST_INSTALL_DIR="@CMAKE_INSTALL_PREFIX@"
```

Comment:
This one is a bit curious. First, [`hl_api_helper.py` line 447ff.](https://github.com/nest/nest-simulator/blob/master/pynest/nest/lib/hl_api_helper.py#L447) check if `NEST_INSTALL_DIR` is set in the environment and then [line 396](https://github.com/nest/nest-simulator/blob/master/pynest/nest/lib/hl_api_helper.py#L394) just uses the internally set docdir from within sli. My suggestion is to remove the pertaining check from `hl_api_helper.py` and hope that this does not break anything.